src/software/simulation/physics/physics_robot.h

```diff
@@ -182,6 +182,13 @@ class PhysicsRobot
      */
     void setPosition(const Point &position);
 
+    /**
+     * Applies the given force vector to the robot at its center of mass
+     *
+     * @param force The force to apply
+     */
+    void applyForceToCenter(const Vector &force);
```

Comment:
Center implies center of the volume to me, so it's a bit confusing
```suggestion
    void applyForceToCenterOfMass(const Vector &force);
```

Reply:
done